runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java

```diff
@@ -43,11 +43,22 @@ default boolean enableTooltips()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "enableColorBasedSkillOrbs",
+		name = "Progress arc color based on skill",
+		description = "Change the color of the progress arc in the xp orb based on skill being trained.",
```

Comment:
It might be good to explicitly state that this overrides the "Progress arc color" option. That might help cut down on support questions on this in the future, if added.